src/sinks/elasticsearch/integration_tests.rs

```diff
@@ -281,6 +281,23 @@ async fn insert_events_over_http() {
     .await;
 }
 
+#[tokio::test]
+async fn insert_events_over_http_with_compression() {
```

Comment:
Nit, same for the other function name.


```suggestion
async fn insert_events_over_http_with_gzip_compression() {
```